### PR TITLE
Rearrange changelog for v0.22

### PR DESCRIPTION
**Pull request by hoodmane**

### CODE REVIEW

File: docs/project/changelog.md [hunk 1]
```diff
@@ -14,81 +14,67 @@ substitutions:
 
 ## Unreleased
 
-- `pyodide-cdn2.iodide.io` is not available anymore. Please use `https://cdn.jsdelivr.net/pyodide` instead.
+- {{ Enhancement }} Updated Emscripten to version 3.1.27
+  {pr}`2958`, {pr}`2950`, {pr}`3027`, {pr}`3107`, {pr}`3148`, {pr}`3236`,
+  {pr}`3239`, {pr}`3280`, {pr}`3314`
+
+### Project Management
```
Comment (rth) [inline docs/project/changelog.md:21]: Maybe
```
## Deployment and testing
```
I have personally never seen "Project Management" as a type of changes in a software changelog, particularly regarding changes that would impact users )

**Review by rth — APPROVED** [review]
Thanks!

File: docs/project/changelog.md (hunk 1 above)
Comment (hoodmane) [reply docs/project/changelog.md:21]: Yeah I just had no idea what to call it =)

**Comment by rth** [comment]
Maybe merged too early as this makes the doc generation fail, also on main. Not very clear what the problem is https://readthedocs.org/projects/pyodide/builds/18958595/

**Comment by hoodmane** [comment]
I think it's not about this PR, there's been ongoing flakiness in readthedocs.

**Comment by hoodmane** [comment]
It seems like readthedocs is usually failing on branches but I haven't seen it fail on main.

**Comment by hoodmane** [comment]
It seems like readthedocs is usually failing on branches but I haven't seen it fail on main.

**Pull request merged**